packages/core/upload/admin/src/components/BulkMoveDialog/BulkMoveDialog.js

```diff
@@ -0,0 +1,135 @@
+    onClose();
+  };
+
+  return (
+    <ModalLayout onClose={handleClose} labelledBy="title">
+      <ModalHeader>
+        <Typography fontWeight="bold" textColor="neutral800" as="h2" id="title">
+          {formatMessage({
+            id: getTrad('modal.folder.move.title'),
+            defaultMessage: 'Move elements to',
+          })}
+        </Typography>
+      </ModalHeader>
+
+      <ModalBody>
+        <Formik
+          validateOnChange={false}
+          onSubmit={handleSubmit}
+          initialValues={initialFormData}
+          initialErrors={initialErrors}
+        >
+          {({ values, errors, setFieldValue }) => (
+            <Form noValidate>
+              <Grid gap={4}>
+                <GridItem xs={12} col={12}>
+                  <Stack spacing={1}>
+                    <FieldLabel htmlFor="folder-destination">
+                      {formatMessage({
+                        id: getTrad('form.input.label.folder-location'),
+                        defaultMessage: 'Location',
+                      })}
+                    </FieldLabel>
+
+                    <SelectTree
+                      options={folderStructure}
+                      onChange={value => {
+                        setFieldValue('destination', value);
+                      }}
+                      defaultValue={values.destination}
+                      name="destination"
+                      menuPortalTarget={document.querySelector('body')}
+                      inputId="folder-destination"
+                      {...(errors.destination
+                        ? {
+                            'aria-errormessage': 'folder-destination-error',
+                            'aria-invalid': true,
+                          }
+                        : {})}
+                    />
+
+                    {errors.destination && (
+                      <Typography
+                        variant="pi"
+                        as="p"
+                        id="folder-destination-error"
+                        textColor="danger600"
+                      >
+                        {errors.destination}
+                      </Typography>
+                    )}
+                  </Stack>
+                </GridItem>
+              </Grid>
+
+              <VisuallyHidden>
+                <button type="submit" tabIndex={-1} ref={submitButtonRef} name="hidden-submit">
+                  {formatMessage({ id: 'submit', defaultMessage: 'Submit' })}
+                </button>
+              </VisuallyHidden>
+            </Form>
+          )}
+        </Formik>
+      </ModalBody>
+
+      <ModalFooter
+        startActions={
+          <Button onClick={handleClose} variant="tertiary" name="cancel">
+            {formatMessage({ id: 'cancel', defaultMessage: 'Cancel' })}
+          </Button>
+        }
+        endActions={
+          <Button onClick={() => submitButtonRef.current.click()} name="submit" loading={isLoading}>
```

Comment:
works well but wondering if we could just wrap ModalHeader, Body and Footer in the Form so we don't have to add a VisuallyHidden button?
found [other places](https://github.com/strapi/strapi/blob/master/packages/core/admin/admin/src/content-manager/pages/ListSettingsView/components/EditFieldForm.js#L50) where we did it, WDYT?

Reply:
I think I copied the idea from there. Basically, the problem is, that we'd need to wrap the form and the `ModalFooter` inside of the form, which didn't seem worth exploring to me.

Do you think we should try that?

Reply:
I think I didn't get the link to the reference you were talking about!
I know we wrap the `ModalHeader` and `ModalFooter` inside the form in the CM (as the link I sent), so it's a pattern we already have, I think we could do the same here!

Reply:
Done in ec42375ee4d875a2b085f5f029776371534fe4c1